app/src/main/java/org/hwyl/sexytopo/model/graph/Coord2D.java

```diff
@@ -27,10 +27,30 @@ public Coord2D minus(Coord2D coord2D) {
         return new Coord2D(this.x - coord2D.x, this.y - coord2D.y);
     }
 
+    public Coord2D multiply(Coord2D coord2D) {
+        return new Coord2D(this.x * coord2D.x, this.y * coord2D.y);
+    }
+
+    public Coord2D div(double divisor) {
+        return new Coord2D(this.x / divisor, this.y / divisor);
+    }
+
     public Coord2D scale(double scale) {
         return new Coord2D(x * scale, y * scale);
     }
 
+    public double mag() {
+        return Math.sqrt(this.x * this.x + this.y * this.y);
+    }
+
+    public Coord2D normalize() {
```

Comment:
SexyTopo is written in British English, old chap. One would normally write `normalise()`.

Reply:
Even though you've got other copy-paste Normalize() methods in here ;)